src/qibolab/platform.py

```diff
@@ -26,7 +25,13 @@ class Platform:
     """
 
     def __init__(self, name, runcard, instruments, channels):
-        log.info(f"Loading platform {name} from runcard {runcard}")
+        # TODO: convert to single log. This is just to avoid to
+        # print the all dictionary
+        if name == "dummy":
+            log.info(f"Loading platform {name}")
+        else:
+            log.info(f"Loading platform {name} from runcard {runcard}")
+
```

Comment:
If it's dummy, just make the runcard `None`.

Reply:
It's a special case anyhow (at least for the time being).

The only other option (that I'd actually like a lot), it's to always pass a dictionary, instead of a path.

Reply:
Actually this is not that trivial because in the case of `dummy` runcard is the dictionary.
I duplicated to avoid printing the all dictionary when allocating the platform.
For sure there is a better way to do it.

Reply:
If you like this approach (dict + function to create platform on the same file). We could even adopt it for all the platforms until the db is ready.

Reply:
If we can avoid passing a runcard path down below, so much the better :)

(hopefully at some point we'll be able to pass objects, avoiding even `dict`s)

Reply:
Actually at this point since we are disentangling runcards from qibolab I suggest to have only
```py
log.info(f"Loading platform {name}")
``` 
as log.

Reply:
Even fine, but at some point the *runcard* will have to be loaded in any case (from file or db). We could log at that point.